gramps/gen/plug/_gramplet.py

```diff
@@ -127,15 +128,20 @@ def get_active(self, nav_type):
         """
         Return the handle of the active object for the given navigation type.
         """
-        return self.uistate.get_active(nav_type, self.nav_group)
+        handle = self.uistate.get_active(nav_type, self.nav_group)
+        if handle and nav_type in self.dbstate.db.get_table_names() and \
```

Comment:
I know it was under review ("to be added if and when needed"); it turns out to be a useful function in this case.  I could code around it with a helper that directs the 'nav_type' to the various 'has_???_handle' methods, but that seems a bit inelegant since both db types then collapse them to another (hidden) single helper method method again.